tenant-partitioning-config-service-impl/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+plugins {
+  `java-library`
+  jacoco
+  id("org.hypertrace.jacoco-report-plugin")
+}
+
+dependencies {
+  api(projects.tenantPartitioningConfigServiceApi)
+  implementation(projects.configServiceApi)
```

Comment:
looks like there's likely some copy paste in this dependency list. This is independent of the config service right? Please remove anything not being used.

Reply:
done